### ISSUE TITLE
zj zk issue

### ISSUE THREAD

**Issue by a-sk** [spiiph/vim-space#12]
Hi, i have strange vim-space behavior about repeating `zj zk` commands.
It produces only **zj** movement doesnt matter if i pushed **zj** or **zk**.

commit: 9440e388cc75754ab7c58441f3ef965a70a411db
distr:      MacVim

**Comment by spiiph** [comment]
Yes, I believe this is a bug I found not too long ago. I will check if I have committed a fix, or otherwise do so.
